src/assets/style/command.css

```diff
@@ -1,6 +1,5 @@
-
 #command {
-  background-color: rgba(0,0,0,0.7);
+  background-color: #141414;
```

Comment:
Also, I discovered today you can do `$('#elementId').css({ color: 'white', background-color: 'black'});` sort of thing. Though you want jQuery to die. Just wasn't sure if you knew that was a thing (you probably do)...